whisper/tokenizer.py

```diff
@@ -29,6 +29,7 @@
     "fi": "finnish",
     "vi": "vietnamese",
     "iw": "hebrew",
```

Comment:
left the previous code here for backwards compatibility, but the LANGUAGE_TO_CODE now returns `he` as it should